modules/references/client/components/read-references/ReferencesReadPresentational.js

```diff
@@ -0,0 +1,93 @@
+import { useTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
+import React from 'react';
+import Reference from './Reference';
+
+export default function ReferencesReadPresentational({
```

Comment:
I think I'd like to combine this `Presentational` component to `ListReferences.component` for clarity.